basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -820,6 +796,7 @@ public static String mapInboundRole(String incomingRoles, Set<String> siteRoles,
 
 			// Loop through Learner,Student,access
 			for (String sakaiRole : sakaiRoleList) {
+				if ( "Ignore".equalsIgnoreCase(sakaiRole) ) continue;
```

Comment:
I don't see used anymore, seems like it can be removed too, unless you wanted to leave it there as an unused feature. Though, removing just seems the best.

Reply:
I am going to leave it in - because there are so many override possibilities - I want a way to shut a rule off.